src/aegis_ai_web/src/main.py

```diff
@@ -62,7 +62,20 @@ async def dispatch(self, request: Request, call_next):
     # middleware to add GSSAPI authentication
     app.add_middleware(GSSAPIMiddleware, spn=kerberos_spn)
 
-logfire.instrument_fastapi(app)
+# middleware enabling CORS
+cors_target_url = os.getenv("AEGIS_CORS_TARGET_URL", "http://localhost:5173")
```

Comment:
Please document the variable here: https://github.com/RedHatProductSecurity/aegis-ai/blob/main/docs/env-vars.md